### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by kodiultimate**
Hi,

If you are happy with the files please feel free to push it  :) thanks

### CODE REVIEW

File: resources/language/resource.language.pl_pl/strings.po [hunk 1]
```diff
@@ -0,0 +1,2253 @@
...
+
+#: /resources/lib/downloader.py
+msgctxt "#32060"
+msgid "files extracted"
+msgstr "pliki rozpakowane"
+
+#: /resources/lib/player.py
+msgctxt "#32061"
+msgid "Play with"
+msgstr " "
```
Comment (jurialmunkey) [inline resources/language/resource.language.pl_pl/strings.po]: Is this space a typo? Should either translate string or leave as original `""`

**Review by jurialmunkey — CHANGES_REQUESTED** [review]
The "Play with" string you've put as a space `" "` - should either be translated or left as original `""`

Fix that and I'll merge.

**Comment by kodiultimate** [comment]
Oh yes, it's a typo! will fix that asap

**Comment by jurialmunkey** [comment]
Thanks so much!

**Pull request merged**